library/CMakeLists.txt

```diff
@@ -68,6 +68,7 @@ set(src_crypto
     psa_crypto_cipher.c
     psa_crypto_client.c
     psa_crypto_driver_wrappers.c
+    psa_crypto_driver_wrappers_no_static.c
```

Comment:
line 70 and line 192 should be removed for the file is included in `psa_crypto.c`

Reply:
I disagree: a `.h` file should only contain declarations, macro/type definitions, and inline function definitions, not linkable variable/function definitions. Since this file defines functions, it is not a header file and should not be called `*.h`.

I'm ok with either calling it `*.c` or using a different extension.

We have prior history of `#include` on non-header files:

* `programs/psa/psa_constant_names_generated.c`
* `*.function` in the test framework

Reply:
Got it. 
But It looks like not reply for this comment.  I still think `psa_crypto_driver_wrappers.c` should be removed from source file list , it might cause duplicated symbol issues in link stage.

Reply:
Indeed, I was replying to

> psa_crypto_driver_wrappers.c should renamed to header file(.h) 

and not to this comment specifically.

Reply:
I would strongly prefer that files that get `#include`d do not have a `.c` extension. OpenSSL uses `.inc`, but I am happy with other suggestions. `.function` files are only used in the test framework; using that extension here might be confusing.

For `static inline` functions we have a couple of `mbedtls_foo__impl.h` files.

Reply:
Agree with @tom-cosgrove-arm , it should use a different extension.